template-parser/components/templateHandlers.js

```diff
@@ -39,13 +39,13 @@ module.exports.cppHandler = async (templateFilepath, outputFilepath, data) => {
     }).join("");
 
     let replacer = (match) => {
-      return `#ifdef NEW_CODE\n${paramsFormatted}#else // NEW_CODE`
+      return `\/\/ BEG_CODE_OPTIONS\n${paramsFormatted}\/\/ END_CODE_OPTIONS`
```

Comment:
Great, I see that you removed the #ifdef in the trueblocks-core code, and changed this in the 2 places it needed to be changed. Thanks.